modules/EPMC.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+from pyspark import *
+from pyspark.sql import *
+from pyspark.sql.types import *
+from pyspark.sql.functions import *
+from pyspark.sql.window import Window
+import logging
+
+
+# example run local, make sure you dont pass many parquet files as input to not run out of mem
+# also, double check you put the ram size properly in the local settings in the SparkConf map
+# python modules/EPMC.py --local \
+#        --cooccurrenceFile /home/mkarmona/src/opentargets/data/platform/epmc/epmc-cooccurrences \
+#        --outputFile test_out
+#        --logFile epmc_parser.log
+
+
+def main():
+
+    ##
+    ## Parsing parameters:
+    ##
+    parser = argparse.ArgumentParser(description='This script generates target/disease evidence strings from ePMC cooccurrence files.')
+    parser.add_argument('--cooccurrenceFile', help='Partioned parquet file with the ePMC cooccurrences', type=str, required=True)
+    parser.add_argument('--outputFile', help='Resulting evidence file saved as compressed JSON.', type=str, required=True)
+    parser.add_argument('--logFile', help='Destination of the logs generated by this script.', type=str, required=False)
+    parser.add_argument('--local', help='Destination of the logs generated by this script.', action='store_true', required=False, default=False)
+    args = parser.parse_args()
```

Comment:
Judgement call; I would move the parser definition from here into the `if __name__ == '__main__'` block. The reason is, when `parse_args()` is called in the main function, it makes it impossible to import from this module to anywhere. While we're not necessarily planning to import this `main()` function from anywhere, I think it's better for consistency to make it always importable

Reply:
That's a really good point. I was also thinking about it earlier that we are kind of defying the whole point of the `if __name__ == "__main__":` condition. Although so far it is only your parser that follows this pattern, we should move into this direction.

Reply:
Also if we keep "importability" in mind, the initialization of the logger should also be in the `if` block, and potentially the initiation of the spark session as well. We discuss it later.